ldi-core/ldes-client/request-executor/src/main/java/ldes/client/requestexecutor/domain/valueobjects/Response.java

```diff
@@ -27,8 +32,10 @@ public Optional<String> getBody() {
 		return Optional.ofNullable(body);
 	}
 
-	public Optional<String> getValueOfHeader(final String key) {
-		return Optional.ofNullable(headers.get(key));
+	public Optional<String> getFirstHeaderValue(final String key) {
```

Comment:
not really a problem.. but do we expect a duplicate key? since headers act like maps, we only expect one entry no?
So maybe "First" is then not needed ?

Reply:
https://stackoverflow.com/questions/4371328/are-duplicate-http-response-headers-acceptable